src/components/edit/EditServicesTable.tsx

```diff
@@ -115,9 +107,8 @@ export const EditServicesTable: React.FC = () => {
     });
 
     // TODO: Should refetch data
-
-
-    return { ...row.data.createService, isNew: false };
+    setRows(prev => [...prev, { ...row.data.createdService, isNew: false }]);
```

Comment:
wait sorry, why'd you add this in :face_with_spiral_eyes: `createdService` is undefined... 

Looking at this also made me notice that this PR has another bug, where a _new_ service cannot be created without a price even though price is optional... but if you feel like you'll cry, we can merge this first and do the submitjob PR and then come back and fix this bug :zany_face: